doc/api/typescript.md

```diff
@@ -172,14 +172,22 @@ To discourage package authors from publishing packages written in TypeScript,
 Node.js will by default refuse to handle TypeScript files inside folders under
 a `node_modules` path.
 
+### Paths aliases
+
+[`tsconfig` "paths"][] won't be transformed and therefore produce an error. The closest
+feature available are [subpath imports][] with the limitation that they need to start
```

Comment:
```suggestion
feature available is [subpath imports][] with the limitation that they need to start
```

(I think this is grammatically correct? otherwise `are` mismatches `closest feature` being singular...)